diff --git a/node_modules/ani-cursor.js/ani-cursor.js b/node_modules/ani-cursor.js/ani-cursor.js
index a54e327..f8052e8 100644
--- a/node_modules/ani-cursor.js/ani-cursor.js
+++ b/node_modules/ani-cursor.js/ani-cursor.js
@@ -1,5 +1,6 @@
 import { Buffer } from "buffer";
-import { RIFFFile } from "riff-file";
+import * as riff from 'riff-file';
+const RIFFFile = riff.RIFFFile;
 class ANIMouse {
   LoadedANIs = [];
   URLPathReg = /[^a-zA-Z0-9-]+/g;
